index.d.ts

```diff
@@ -125,6 +125,10 @@ declare module "hyperapp" {
     effecter: (dispatch: Dispatch<S>, payload: P) => void | Promise<void>,
     payload: P
   ]
+  
+  // Effects can be declared conditionally
```

Comment:
I know I'm being picky but let's put a period at the end of the sentence. 😄

Reply:
Fixed in force-push